examples/integration-tutorials/using-the-analytics-dashboard/using_the_analytics_dashboard.bal

```diff
@@ -0,0 +1,244 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/io;
+
+http:Client healthcareServiceEP = new("http://localhost:9090/healthcare");
```

Comment:
Yes. this scenario seems much like a test. I can change it so that the user just has to run the test. 

Moreover this would also test each endpoint in the Healthcare service. I will change it so that it is a test case.